kafka/src/main/java/io/specmesh/kafka/Provisioner.java

```diff
@@ -127,4 +133,82 @@ static ParsedSchema getSchema(final String topicName, final String schemaRef, fi
         }
         throw new RuntimeException("Failed to handle topic:" + topicName + " schema: " + path);
     }
+
+    /**
+     * Provision Topics, ACLS and schemas
+     *
+     * @param apiSpec
+     *            given spec
+     * @param schemaResources
+     *            schema path
+     * @param adminClient
+     *            kafka admin client
+     * @param schemaRegistryClient
+     *            sr client
+     * @throws ExecutionException
+     *             when cant provision topics
+     * @throws InterruptedException
+     *             process interrupted
+     * @throws TimeoutException
+     *             stalled
+     */
+    public static void provision(final KafkaApiSpec apiSpec, final String schemaResources,
+            final AdminClient adminClient, final SchemaRegistryClient schemaRegistryClient)
+            throws ExecutionException, InterruptedException, TimeoutException {
+        provisionTopics(apiSpec, adminClient);
+        provisionSchemas(apiSpec, schemaResources, schemaRegistryClient);
+        provisionAcls(apiSpec, adminClient);
+
+    }
+
+    /**
+     * setup sasl_plain auth creds
+     *
+     * @param principle
+     *            user name
+     * @param secret
+     *            secret
+     * @return client creds map
+     */
+    public static Map<String, Object> clientAuthProperties(final String principle, final String secret) {
+        return Map.of("sasl.mechanism", "PLAIN", AdminClientConfig.SECURITY_PROTOCOL_CONFIG, "SASL_PLAINTEXT",
+                "sasl.jaas.config", String.format("org.apache.kafka.common.security.plain.PlainLoginModule required "
+                        + "username=\"%s\" password=\"%s\";", principle, secret));
+
+    }
+
+    /**
+     * Configure environment for SASL_PLAIN auth with 2 sets of users
+     *
+     * @param domainUser
+     *            owner user
+     * @param domainSecret
+     *            their secret
+     * @param otherDomainUser
+     *            other user
+     * @param otherDomainSecret
+     *            their secret
+     * @return env map for broker
+     */
+    public static Map<String, String> testAuthorizerConfig(final String domainUser, final String domainSecret,
+            final String otherDomainUser, final String otherDomainSecret) {
+        final Map<String, String> env = new LinkedHashMap<>();
+        env.put("KAFKA_AUTO_CREATE_TOPICS_ENABLE", "false");
+        env.put("KAFKA_ALLOW_EVERYONE_IF_NO_ACL_FOUND", "true");
+        env.put("KAFKA_AUTHORIZER_CLASS_NAME", "kafka.security.authorizer.AclAuthorizer");
+        env.put("KAFKA_SUPER_USERS", "User:OnlySuperUser");
```

Comment:
Do we need a super user?  If not, remove.

Reply:
ok